### ISSUE TITLE
Logout state can not be found

### ISSUE THREAD

**Issue by DaveArkley-Codiance** [DuendeSoftware/Support#545]
**Which version of Duende IdentityServer are you using?**

Starting Duende IdentityServer version "6.2.0+d923c6c10f11013223f6998059c20f2ac26ba44a" (".NET 6.0.14")

**Which version of .NET are you using?**

.Net 6

**Describe the bug**

When a client application calls Logout I am passed a logoutID as expected and this is used to retrieve a logoutcontext. The returned
context always has the PostLogoutRedirectUri property set to null. I can not see anything that looks like the state value in any other properties on the context.

**To Reproduce**

Steps to reproduce the behavior.

**Expected behavior**

That I can access the state variable and pass it in the returnUrl to the calling client application

**Additional context**

Client startup contains:

		builder.Services.AddAuthentication(options =>
			{
				options.DefaultScheme = "Cookies";
				options.DefaultChallengeScheme = "OpenIdConnect";

			}).AddCookie("Cookies")
			.AddOpenIdConnect("OpenIdConnect", opt =>
			{
				opt.SignInScheme = "Cookies";
				opt.Authority = identityServerOptions.AuthorityUri;
				opt.ClientId = identityServerOptions.WebClientName;
				opt.ResponseType = "code";
				opt.UsePkce = true;
				opt.ResponseMode = "form_post";
				opt.SaveTokens = true;
				opt.ClientSecret = identityServerOptions.WebClientSecret;
				opt.GetClaimsFromUserInfoEndpoint = true;
				opt.Scope.Add("--elided--");
				opt.ClaimActions.MapUniqueJsonKey("--elided--", "--elided--");

				opt.TokenValidationParameters = new TokenValidationParameters
				{
					RoleClaimType = "role"
				};
			});
			.

**Comment by josephdecock** [comment]
Check if the client is configured in identity server to allow the given post logout redirect uri. Also, FYI, IdentityServer logs quite a lot of error details. You probably have log warnings like "Invalid PostLogoutRedirectUri: {postLogoutRedirectUri}".

**Comment by DigitalSynthesis** [comment]
Yes the client is configured correctly to allow the post redirect  url and indeed the client application is redirected to the required page after logout, but the state is not reurned.

**Comment by josephdecock** [comment]
Check the IdentityServer logs during logout. There likely is more information there that will be helpful.

**Comment by josephdecock** [comment]
Any update here? Can we close this issue?

**Comment by DigitalSynthesis** [comment]
I didn't resolve this problem, but the requirement has gone away. Thanks for everyone's suggestions, and yes Joseph I'm happy if you close it.
